src/app/components/shared/timezone/timezone.component.ts

```diff
@@ -0,0 +1,58 @@
+import { Component, Input, OnChanges } from "@angular/core";
+import { DateTimeTimezone } from "@interfaces/apiInterfaces";
+import {
+  AbstractModel,
+  isUnresolvedModel,
+  UnresolvedModel,
+} from "@models/AbstractModel";
+import { Site } from "@models/Site";
+
+@Component({
+  selector: "baw-timezone",
+  template: `
+    <span [ngbTooltip]="timezone">
+      {{ formattedDate }}
+    </span>
+  `,
+  styles: [
+    `
+      span {
+        text-decoration: underline dotted;
+        cursor: help;
+        text-decoration-skip-ink: none;
+      }
+    `,
+  ],
+})
+export class TimezoneComponent implements OnChanges {
+  @Input() public dateTime: DateTimeTimezone;
```

Comment:
I think this should just be called `value` - like any other input we'd see. The name of the component already explains what it does.